beta/examples/tensorflow/object_detection/main.py

```diff
@@ -197,13 +200,13 @@ def train(train_step, test_step, eval_metric, train_dist_dataset, test_dist_data
             train_metric_result.update({'learning_rate': optimizer.lr(optimizer.iterations).numpy()})
 
             train_summary_writer(metrics=train_metric_result, step=optimizer.iterations.numpy())
+            time = timer.toc(average=False)
 
             if step % 100 == 0:
-                logger.info('Step {}/{}'.format(step, steps_per_epoch))
+                logger.info('Step: {}/{} Time: {:.3f}'.format(step, steps_per_epoch, time))
```

Comment:
Could you clarify what time you measure? Is it wall time for 100 steps or average time for step?

Reply:
> Please, specify what units the time is logged. Is it seconds?

Yes, it is seconds.

> Could you clarify what time you measure? Is it wall time for 100 steps or average time for step?

It is time for one current step.

For clarity, I suggest use

**Average time per batch** - average processing time per batch
**Total elapsed time** - time elapsed since the start

Reply:
@alexsu52 What do you think of that?

Reply:
@andrey-churkin I think we can leave **Time**, but instead of the time of one current step, output the elapsed time.

Reply:
Done

Reply:
It looks like you report time for one step not elapsed time. Could you check?